odl/space/cu_ntuples.py

```diff
@@ -1453,10 +1444,7 @@ def norm(self, x):
         norm : `float`
             The norm of the vector
         """
-        if self.exponent == 2.0:
-            from math import sqrt
-            return sqrt(self.const) * float(_norm_default(x))
-        elif self.exponent == float('inf'):  # Weighting irrelevant
+        if self.exponent == float('inf'):  # Weighting irrelevant
```

Comment:
The comment is not valid any more, I changed the behavior after our discussions. Why `if` instead of `elif`?